src/Lessons-api.Data/Models/LessonEntity.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+
+namespace Lessons_api.Data.Models
+{
+    public class LessonEntity
+    {
+        public LessonEntity()
+        {
+            StudentLessonRels = new HashSet<StudentLessonRel>();
+        }
+
+        public int Id { get; set; }
+        public int TeacherId { get; set; }
+        public string Title { get; set; }
+        public string Description { get; set; }
+        public string Price { get; set; }
+        public string Language { get; set; }
+        public ICollection<StudentLessonRel> StudentLessonRels { get; set; }
```

Comment:
You can also add Collection of Students here. 
Please check [this link](https://docs.microsoft.com/en-us/ef/core/modeling/relationships?tabs=fluent-api%2Cfluent-api-simple-key%2Csimple-key) for reference